js2-mode.el

```diff
@@ -7264,7 +7264,7 @@ are ignored."
            when (and (eq 'block (js2-comment-node-format node))
                      (save-excursion
                        (goto-char (js2-node-abs-pos node))
-                       (looking-at "/\\*global ")))
+                       (looking-at "/\\* ?+global ")))
```

Comment:
`" *"` would match any characters, including other words, in front of `global`.  If you have non space characters before `global` it shouldn't be considered valid.